spec/factories/session.rb

```diff
@@ -0,0 +1,124 @@
+FactoryBot.define do
+  factory :full_session, class: Hash do
```

Comment:
I think you might be right. I guess the question is what counts as a "basic" session: one where the client is asylum-supported so only has to enter about 3 screens-worth of information? Or one where they've filled out every screen but entered "no"/0 to everything? Or even one where it's not got much in it yet because it's incomplete? I guess there are lots of reasons why a session might have less than every possible value in it. I wonder if ultimately we'd want a fixture for each of them, named for that reason. Definitely open to ways of making the tests more expressive.